AcademyChallenge/Domain/Repos.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Repos.swift
+//  AcademyChallenge
+//
+//  Created by Keven Esmael on 17/10/2022.
+//
+
+import Foundation
+
+import UIKit
+
+
+struct Repos: Codable {
+	let full_name: String
```

Comment:
in Swift, by convention, we always use camelCase to name our variables. this variable should be named `fullName` instead of `full_name` (that is snake_case). To convert from the names that is on the API and the names you have on your properties you can use the CodingKeys mapping

Reply:
solved